paddle/fluid/inference/api/paddle_analysis_config.h

```diff
@@ -178,7 +178,7 @@ struct AnalysisConfig {
 
   /** Turn on MKLDNN.
    */
-  void EnableMKLDNN();
+  void EnableMKLDNN(int reuse_id = 0);
```

Comment:
reuse_id-》mkldnn_reuse_id
And Please add comment for mkldnn_reuse_id.

Reply:
done